webpack.config.js

```diff
@@ -1,31 +1,14 @@
 /* global __dirname */
 
 const process = require('process');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
-const webpack = require('webpack');
 
 const minimize
     = process.argv.indexOf('-p') !== -1
         || process.argv.indexOf('--optimize-minimize') !== -1;
-const plugins = [
-    new webpack.LoaderOptionsPlugin({
-        debug: !minimize,
-        minimize
-    })
-];
-
-if (minimize) {
-    plugins.push(new webpack.optimize.ModuleConcatenationPlugin());
-    plugins.push(new UglifyJsPlugin({
-        cache: true,
-        extractComments: true,
-        parallel: true,
-        sourceMap: true
-    }));
-}
 
 const config = {
     devtool: 'source-map',
+    mode: minimize ? 'development' : 'production',
```

Comment:
@saghul @virtuacoplenny Shouldn't this be exactly the opposite? I think at the moment when you pass -p we will be in development mode.

Reply:
Looks like I am, indeed and idiot. You're totally right!